src/controller/userController.js

```diff
@@ -47,6 +48,77 @@ class UserController {
       });
     }
   }
+
+  /**
+  * List Authors
+  * Route: GET: /authors
+  * Route: GET: /authors/:search
+  * @param {object} req -Request object
+  * @param {object} res - Response object
+  * @return {res} res - Response object
+  * @memberof USerController
+ */
+  static async getAuthors(req, res) {
+    try {
+      let authors;
+      if (req.params.search) {
+        authors = await Users.findAll({
+          attributes: [
+            'id',
+            'firstName',
+            'lastName',
+            'email',
+            'image',
+            'bio',
+            'username'
+          ],
+          where: {
+            [Op.or]: [
+              {
+                username: {
+                  [Op.like]: `%${req.params.search}%`
+                }
+              },
+              {
+                firstName: {
+                  [Op.like]: `%${req.params.search}%`
+                }
+              },
+              {
+                lastName: {
+                  [Op.like]: `%${req.params.search}%`
+                }
+              },
+            ],
+          }
+        });
+      } else {
+        authors = await Users.findAll({
+          attributes: [
+            'id',
+            'firstName',
+            'lastName',
+            'email',
+            'image',
+            'bio',
+            'username'
+          ],
+          where: {
+            isVerified: true,
+          }
+        });
+      }
+      res.status(200).json({
```

Comment:
You response message, to me, is not self-explanatory. Something like "Request completed successfully" looks more explanatory.